CHANGELOG.md

```diff
@@ -3,6 +3,79 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.7
+
+_Nov 4, 2021_
+
+- 👁 Move virtualization logic to hook (#3079) @m4theushw
+- 🌍 Add hebrew localization (he-IL) (#3028) @ColdAtNight
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.0.0-beta.7` / `@mui/x-data-grid-pro@v5.0.0-beta.7`
+
+#### Breaking changes
+
+- [core] Prefix selectors from `useGridContainerProps` with `unsafe` (#3002) @flaviendelangle
+
+  We are temporarily making the dimension APIs of the grid private to be able to clean them in a future minor release. The current approach is causing a lot of useless re-renders.
+  Please reach to us if you are really using any of those selectors to help us understand what are the community needs around this subject.
```

Comment:
No "us"

```suggestion
  If you relay on any of these selectors, open an issue explaining the use case so that will be taken into account when refactoring them.
```